_posts/en/2021-12-10-iitb-diagnostic-study.md

```diff
@@ -0,0 +1,139 @@
+
+IITB needs to foster a culture of continuous learning, promote professional development, and support employee development to update and improve workforce skills that meet the pace of IM/IT change and evolution.
+
+[Gartner TalentNeuron™](https://www.gartner.com/en/human-resources/research/talentneuron) data shows that the total number of skills required for a single job is increasing by 10% year over year.
+Lack of resources for upskilling and reskilling employees is another impediment that is slowing down productivity during the day IITB employees.
+
+Managing employees in IITB (2,400 employees) in a large organization like ESDC (28,000 employees) is incredibly challenging and during our conversation with directors, they mentioned excessive, rigid, and redundant rules and regulations are a frequent prevention to action and decision-making.
+When there are too many steps and levels of approval to go through, an overly complicated process and governance structure results. Directors openly commented about how complicated simple tasks seem to be because of the complexities involved in carrying out a request/action.
+Many directors indicated current processes and levels of approval do not add value and in fact delay business operations.
+By decreasing the number of governance committees and lessening the levels of approvals would give way to a more effective governance model that would remediate this problem.
+
+The challenges mentioned above lead to a concern for how directors and their employees are impacted.
+Throughout the interviews, it was clear that these challenges correlate with mental health and well-being, feelings of pressure and being overwhelmed or stressed at work.
+Addressing mental health at work is very common in many organizations, especially during the post-pandemic period. Fortunately, ESDC and IITB is taking steps to promote healthier work habits.
+These steps are promoted by having mental health as a regular topic in the IITB newsletter and various communications, managers conduct regular check-ins with employees, and are provided with resources such as the Employee Assistance Program (EAP) for support.
+In addition, ongoing initiatives such as the “Stay Healthy Challenge” also encourage health and well-being in the workplace.
+
+## Conclusion
+
+At the end of the interview sessions, we found ourselves with a massive number of notes and interesting impressions.
+The amount of information we received was positively overwhelming.
+It was no surprise that analyzing and reflecting on the information was challenging; however, it was important that we took the time to capture all the points and concerns the interviewees brought up.
+Our “What We Heard” report provides a comprehensive overview of what we heard throughout the interviews, and although it is lengthy, it provides readers with a detailed analysis of our observations based on what directors said.
+
```

Comment:
Please include something about how senior management are eager to learn more about our findings and plan to use it as a source of evidence.